sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidget.kt

```diff
@@ -35,16 +35,22 @@ open class BitmapWidget @JvmOverloads constructor(
   constructor(bitmap: Bitmap, position: WidgetPosition, marginX: Float, marginY: Float) : this(
     bitmap = bitmap,
     position = WidgetPosition {
-      horizontal = position.horizontal
-      vertical = position.vertical
-      offsetX = marginX
-      offsetY = marginY
+      horizontalAlignment = position.horizontalAlignment
+      verticalAlignment = position.verticalAlignment
+      offsetX = when (position.horizontalAlignment) {
+        WidgetPosition.Horizontal.LEFT, WidgetPosition.Horizontal.CENTER -> marginX
+        WidgetPosition.Horizontal.RIGHT -> -marginX
```

Comment:
Nit: might be good to add some comment explaining the sign change.

Reply:
Added some comments.